R/use_readme.R

```diff
@@ -2,6 +2,7 @@ use_readme <- function(name){
   usethis::use_template("README.Rmd",
                         data = list(Package = name),
                         package = "starters")
+  usethis::use_build_ignore("README.Rmd")
```

Comment:
Does there also need to be .md build ignore added?

Reply:
I think some folks like to submit the README to CRAN but maybe that's not the way 🤔

Reply:
Fair enough then :)